src/Dialect/ONNX/ONNXOpsHelper.cpp

```diff
@@ -42,13 +44,84 @@ Value OnnxBuilder::matmul(Type Y, Value A, Value B) const {
   return b.create<ONNXMatMulOp>(loc, Y, A, B);
 }
 
-Value OnnxBuilder::reshape(Type outputType, Value input, Value shape) const {
-  return b.create<ONNXReshapeOp>(loc, outputType, input, shape);
+Value OnnxBuilder::reshape(
```

Comment:
Could we keep the original `reshape`? We still want to use it where `shape` is an output of another op, and don't want to construct a DinIndexExpr array.

Reply:
Certainly, I am adding it back.